src/api/app/models/project.rb

```diff
@@ -251,6 +258,49 @@ def cleanup_linking_targets
     end
   end
 
+  def update_meta(request_data, params)
+    result = false
+
+    Project.check_target_of_link(name, request_data)
+    Project.update_project_meta_maintains(request_data)
+    new_repo_names = Project.update_project_meta_repositories(self.name, request_data)
+
+    # permission check passed, write meta
+    removedRepositories = []
+    self.repositories.each do |repo|
+      if !new_repo_names[repo.name] && !repo.remote_project_name
+        removedRepositories << repo
+      end
+    end
+
+    return result unless check_and_remove_repositories(params, removedRepositories)
```

Comment:
Same as [here](https://github.com/openSUSE/open-build-service/pull/1102/files#r38821498). Unless you change the method, the return is superfluous.